src/OpenTelemetry.Instrumentation.SqlClient/Implementation/SqlEventSourceListener.netfx.cs

```diff
@@ -183,16 +183,16 @@ private void OnEndExecute(EventWrittenEventArgs eventData)
                     int compositeState = (int)eventData.Payload[1];
                     if ((compositeState & 0b001) == 0b001)
                     {
-                        activity.SetStatus(Status.Unset);
+                        activity.SetStatus(ActivityStatusCode.Unset);
```

Comment:
we dont need this either.

Reply:
removed it, too; thanks!

Reply:
its still here.... :D

Reply:
Ah, there's an update, please refer to this conversation:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/3751/files/6474e650ac9214e2d26aaa09182d5a75c2d1b189#r992807846